locales/en.json

```diff
@@ -1287,7 +1285,6 @@
     "InvalidToken": "Token is invalid or expired",
     "InvalidUserName": "User name is invalid, can only contain letters or digits",
     "LoginAlreadyAssociated": "A user with this login already exists",
-    "PasswordMismatch": "Passwords must match",
```

Comment:
Don't remove it, it's Identity error currently in use, it just probably should have other text. @NaMax66 write details please